src/Billing/Controllers/StripeController.cs

```diff
@@ -235,14 +240,18 @@ public async Task<IActionResult> PostWebhook([FromQuery] string key)
             }
 
             var subscription = await _stripeFacade.GetSubscription(invoice.SubscriptionId);
+            var customer = await _stripeFacade.GetCustomer(subscription.CustomerId);
 
             if (subscription == null)
             {
                 throw new Exception(
                     $"Received null Subscription from Stripe for ID '{invoice.SubscriptionId}' while processing Event with ID '{parsedEvent.Id}'");
             }
 
-            if (!subscription.AutomaticTax.Enabled)
+            if (_featureService.IsEnabled(FeatureFlagKeys.PM5766AutomaticTax) &&
```

Comment:
This can be extracted into a variable before line 251 and used for both this and the check on line 264 to avoid making 2 requests to LD for the same flag.